### DISCUSSION TITLE
Welcome to freefolio Discussions!

### DISCUSSION THREAD

**Discussion by jofftiquez** [OSSPhilippines/freefolio#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by artdev-hash** [comment]
Hi I'm **Kim**, a Self taught Developer aspiring to become an Expert in the field of Mobile development. One of the **goals** in the community is to **learn** more, so **I** can **create**.. take action to **build** and **share**, to **connect** with other people.. To give birth to an idea and able to share it with the community, To have a significant or lasting impact on someone or something...
